templates/variables.tftpl

```diff
@@ -0,0 +1,14 @@
+%{ if vm_win_hostname != "" ~}
+variable "${vm_win_hostname}_password" {
```

Comment:
This is just cosmetic, but if possible the variable name should remain snake case to be conform to terraform naming conventions.

Reply:
Solved with upper function change for variable.